File: Official Test Code/src/org/usfirst/frc/team1884/robot/Robot.java
package org.usfirst.frc.team1884.robot;

import org.usfirst.frc.team1884.robot.subsystems.Aimer;
import org.usfirst.frc.team1884.robot.subsystems.GRIP;
import org.usfirst.frc.team1884.robot.subsystems.Proportional;
import org.usfirst.frc.team1884.robot.subsystems.Spike;

import edu.wpi.first.wpilibj.AnalogInput;
import edu.wpi.first.wpilibj.IterativeRobot;
import edu.wpi.first.wpilibj.Joystick;
import edu.wpi.first.wpilibj.RobotDrive;

public class Robot extends IterativeRobot {
	RobotDrive drive;
	Joystick stick;
	Proportional p;
	AnalogInput ultrasonic;

	public void teleopInit() {
		//drive = new RobotDrive(0, 1, 2, 3);
		stick = new Joystick(0);
		Spike.INSTANCE.turnOn();
		// p = new Proportional((1.0 / 96.0), 96);
		// ultrasonic = new AnalogInput(0);
	}

	public void teleopPeriodic() {
		// double output = p.getOutput(ultrasonic.getValue() / 7.2436);
		// System.out.println(output);
		// drive.drive(output, 0);
		//
		Aimer.INSTANCE.align();
	}
}

File: C++ Example Code/Jaguar C++ Robot Code/src/Robot.cpp
#include "WPILib.h"

class Robot: public IterativeRobot {
	CANJaguar *frontLeft;
	CANJaguar *frontRight;
	CANJaguar *backLeft;
	CANJaguar *backRight;

	Joystick *joystick;
	RobotDrive *drive;
public:
	Robot() {
		frontLeft = new CANJaguar(0);
		frontRight = new CANJaguar(1);
		backLeft = new CANJaguar(2);
		backRight = new CANJaguar(3);
		joystick = new Joystick(0);
		drive = new RobotDrive(frontLeft,frontRight,backLeft,backRight);
	}

	void AutonomousPeriodic() {
		drive->Drive(0.25, 0.0);
	}

	void TeleopPeriodic() {
		drive->ArcadeDrive(joystick);
	}
};

START_ROBOT_CLASS(Robot)
